internal/cloud/realm/user_ext_test.go

```diff
@@ -120,7 +126,53 @@ func TestRealmUsers(t *testing.T) {
 				}
 				users, err := client.FindUsers(groupID, testApp.ID, filter)
 				assert.Nil(t, err)
-				assert.Equal(t, []realm.User{email3}, users)
+				assert.Equal(t, 1, len(users))
+				assert.Equal(t, email3.ID, users[0].ID)
```

Comment:
[question] if you nested this test in the one just above here, would you again have access to `email3Disabled`, and could that work here for an assert like `assert.Equal(t, []realm.User{email3Disabled}, users)`?  if not, this is totally fine to leave as is

Reply:
Yeah, that would work